libclc/ptx-nvidiacl/libspirv/workitem/get_sub_group_size.cl

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <spirv/spirv.h>
+_CLC_DEF _CLC_OVERLOAD uint __spirv_SubgroupId();
+_CLC_DEF _CLC_OVERLOAD uint __spirv_NumSubgroups();
+_CLC_DEF _CLC_OVERLOAD uint __spirv_SubgroupMaxSize();
+
+_CLC_DEF _CLC_OVERLOAD uint __spirv_SubgroupSize() {
+  if (__spirv_SubgroupId() != __spirv_NumSubgroups() - 1) {
+    return __spirv_SubgroupMaxSize();
+  } else {
```

Comment:
```suggestion
  }
```
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
Fixed in 41d157de0f0f57c850ecc5a54c57de338a87d9a4.